app/controllers/admin/affiliates_controller.rb

```diff
@@ -7,8 +7,27 @@ class Admin::AffiliatesController < Admin::AdminController
     config.actions.add :field_search
     config.field_search.columns = :id, :name, :display_name, :website
 
+    hidden_columns_regexp = /
+      (_created_at|
+      _updated_at|
+      agency_id|
+      css_properties|
+      content_type|
+      file_name|
+      identifier_domain_name|
+      _image|
+      _json|
+      label|
+      _logo|
+      _mappings|
+      parent_agency_link|
+      parent_agency_name|
+      scope_ids|
+      size|
+      use_extended_header)\z/x
+
     attribute_columns = config.columns.reject do |column|
-      column.association or column.name =~ /(_created_at|_updated_at|agency_id|css_properties|content_type|file_name|_image|json|label|_logo|_mappings|scope_ids|size|use_extended_header|visual_design_json)\z/
```

Comment:
Regex may be too much for this. I suggest to use an array instead and then use something like`HIDDEN_COLUMS.include?(column.name)`